app/src/views/OperationalLearnings/index.tsx

```diff
@@ -1,17 +1,195 @@
+import { useState } from 'react';
+import {
+    Tab,
+    TabList,
+    Container,
+    DateInput,
+    SelectInput,
+    Button,
+} from '@ifrc-go/ui';
 import { useTranslation } from '@ifrc-go/ui/hooks';
 
+import CountrySelectInput from '#components/domain/CountrySelectInput';
+import DisasterTypeSelectInput from '#components/domain/DisasterTypeSelectInput';
+import ExportButton from '#components/domain/ExportButton';
+import KeywordSearchSelectInput from '#components/domain/KeywordSearchSelectInput';
+import RegionSelectInput from '#components/domain/RegionSelectInput';
 import Page from '#components/Page';
 
 import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type Option = {
+    key: string;
+    label: string;
+};
 
-// eslint-disable-next-line import/prefer-default-export
 export function Component() {
     const strings = useTranslation(i18n);
+
+    const options: Option[] = [];
+    const keySelector = (option: Option) => option.key;
+    const labelSelector = (option: Option) => option.label;
+    const [regionValue] = useState<0 | 1 | 2 | 3 | 4 | null>(null);
+    const [countryValue] = useState<number | null>(null);
+    const [disasterTypeValue] = useState<number | null>(null);
+    const [sectorValue] = useState<string | null>(null);
+    const [componentValue] = useState<string | null>(null);
+    const [tabValue, setTabValue] = useState('by-sector');
+
+    const handleTabChange = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
This doesn't seem to be the right way handling tab changes. Let's go through the code base and see how it's handled.